edx/edx-app-ios
# LEARNER-431, UI issues on Register and Sign In screens.

### Description

[LEARNER-431](https://openedx.atlassian.net/browse/LEARNER-431)

Update label text in register screen and adjust the spaces between button and label text in sign in screen.

### How to test this PR
- Go on register screen and verify the labels text
- Go on sign in screen and verify the space between facebook register button and text above and below it.
